main.go

```diff
@@ -105,10 +103,6 @@ func main() {
 	flag.BoolVar(&certSigningDisabled, "disable-cert-signing-support", false, "Disable native certificate signing integration")
 	flag.IntVar(&maxKafkaTopicConcurrentReconciles, "max-kafka-topic-concurrent-reconciles", 10, "Define max amount of concurrent KafkaTopic reconciles")
 	flag.Parse()
-
-	podNamespace = os.Getenv("POD_NAMESPACE")
```

Comment:
Since `POD_NAMESPACE` and `SERVICE_ACCOUNT` env vars are not used anymore anywhere I think this part can be removed as well: https://github.com/banzaicloud/koperator/blob/master/charts/kafka-operator/templates/operator-deployment-with-webhook.yaml#L201-L210

Reply:
Yes, It can be removed. There is a chance for using it in the future. Do you think it should be removed?